src/coverlet.console/Logging/ConsoleLogger.cs

```diff
@@ -9,7 +9,8 @@ namespace Coverlet.Console.Logging
 {
     class ConsoleLogger : ILogger
     {
-        private static readonly object _sync = new object();
+        private static readonly object s_sync = new();
```

Comment:
`s_` is the default that was approved but if you want to change it to make no diff or use a different pattern, let me know.

Reply:
I'm ok with field name update